search-app/src/cmr/search/site/data.clj

```diff
@@ -79,31 +80,49 @@
 (defn-timed get-collection-data
   "Get the collection data from elastic by provider id and tag. Sort results
   by entry title"
-  [context tag provider-id]
-  (let [conditions (query-svc/generate-query-conditions-for-parameters
+  ([context tag provider-id]
+   (get-collection-data context tag provider-id []))
+  ([context tag provider-id granule-conditions]
+   (let [conditions (query-svc/generate-query-conditions-for-parameters
                      context
                      :collection
                      {:tag-key tag
                       :provider provider-id})
-        query (query-model/query {:concept-type :collection
-                                  :condition (gc/and-conds conditions)
-                                  :skip-acls? false
-                                  :page-size :unlimited
-                                  :result-format :query-specified
-                                  :result-features [:granule-counts]
-                                  :result-fields [:concept-id
-                                                  :doi
-                                                  :entry-title
-                                                  :short-name
-                                                  :version-id]})
-        result (query-exec/execute-query context query)
-        {:keys [items granule-counts-map]} result]
-    (sort-by :entry-title
-             (map #(assoc % :granule-count (get granule-counts-map (:concept-id %))) items))))
+         query (query-model/query {:concept-type :collection
+                                   :condition (gc/and-conds conditions)
+                                   :skip-acls? false
+                                   :page-size :unlimited
+                                   :result-format :query-specified
+                                   :result-features [:granule-counts]
+                                   :result-fields [:concept-id
+                                                   :doi
+                                                   :entry-title
+                                                   :short-name
+                                                   :version-id]})
+         result (query-exec/execute-query context query)
+         collection-concept-ids (into [] (map (fn [item] (:concept-id item)) (:items result)))
```

Comment:
I'd suggest simplifying this code to `(vec (map :concept-id (:items result)))`

Reply:
done